recipes-bsp/u-boot/files/upstream/0025-arm-dts-update-flash-layout.patch

```diff
@@ -0,0 +1,88 @@
+From 0413a846eadb1e4549f3a73261b5ca945e239217 Mon Sep 17 00:00:00 2001
+From: Chao Zeng <chao.zeng@siemens.com>
+Date: Thu, 3 Jun 2021 11:45:01 +0800
+Subject: [PATCH 25/25] arm:dts:update flash layout
+
+there is a 16M flash on our board,in order to unify uboot
+for pg1 and pg2, keep the pg1 and pg2 sysfw on the flash at
+the same time.
```

Comment:
Can we refactor this message already to be placed on top of the upstream queue and submitted soon as well?
Eg., subject should reflect what is happening more precisely (add sysfw slots for PG2 device in flash layout).

Also, why do we reserve the whole flash now when we only need to add a few additional MB?

Reply:
in the latest TI SDK, there  have 6 icssg firmwares, reserve the reaming space for the extra icssg firmwares in the future.

Reply:
I think 2 * 512K for sysfw SR2 and 6 * 64K for pru firmware is enough.

Reply:
yes,based on this, extend our flash layout to 9.375M